<def f='linux-5.3.1/include/trace/events/ext4.h' l='2683' macro='1' type='void trace_ext4_error(struct super_block * sb, const char * function, unsigned int line)'/>
<use f='linux-5.3.1/fs/ext4/super.c' l='505' u='c' c='__ext4_error'/>
<use f='linux-5.3.1/fs/ext4/super.c' l='530' u='c' c='__ext4_error_inode'/>
<use f='linux-5.3.1/fs/ext4/super.c' l='566' u='c' c='__ext4_error_file'/>
<use f='linux-5.3.1/fs/ext4/super.c' l='779' u='c' c='__ext4_grp_locked_error'/>
